src/libs/humanizer/modules/Uniswap/uniUniversalRouter.ts

```diff
@@ -31,26 +31,17 @@ const extractParams = (inputsDetails: any, input: any) => {
 }
 
 // '0x1234' => ['0x12', '0x34']
-function parseCommands(commands: string, emitError?: Function): string[] | null {
-  try {
-    if (!commands.startsWith('0x') || commands.length % 2 !== 0) return null
-    const hex = commands.slice(2)
-    const hexRegex = /[0-9A-Fa-f]/g
-    if (!hexRegex.test(hex)) return null
-    const res = hex.match(/.{2}/g)?.map((p: string) => `0x${p}`)
-    return res as string[]
-  } catch (e) {
-    emitError &&
```

Comment:
Wont we have an error anymore?

Reply:
the argument we are gonna pass to this function come from, presumably, uniswap txns' calldata. This is the happy case. If the arg is invalid that means the calldata of the original txn is not uniswaps and this module (Uniswap module calls parseCommands) is not the one for this call. 

Happy case = no error
error case = it is supposed to return null anyway

Reply:
since the humanizer is now pure we wont emit errors